src/adapter.ts

```diff
@@ -276,7 +276,7 @@ export class MongooseAdapter implements BatchAdapter, FilteredAdapter, Updatable
 
     for (const word of [line.v0, line.v1, line.v2, line.v3, line.v4, line.v5]) {
       if (word !== undefined) {
-        lineText = `${lineText},${word}`
```

Comment:
Should check if the word is already wrapped in inverted commas

Reply:
Done